<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order_new</class>
 <widget class="QDialog" name="order_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1721</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>667</y>
     <width>1448</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>121</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>53</y>
     <width>571</width>
     <height>411</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>571</width>
     <height>411</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>571</width>
     <height>411</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbID_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер заказа </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата Заказа</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbName_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>219</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> ФИО Клиента</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbINN_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>298</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО Сотрудника</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>377</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Оплата</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBpay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>377</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>22</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>true</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>false</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>371</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>441</width>
      <height>371</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="leOrder_number"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leDate_2"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbFio_client"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbFio_emp"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>590</y>
     <width>491</width>
     <height>44</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>491</width>
     <height>44</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAdd_2">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit_2">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDel_2">
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRemove_2">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>509</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>351</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>351</width>
     <height>41</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад </string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>871</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>871</width>
     <height>561</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
